src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -363,6 +363,12 @@ public void addSFSessionProperty(String propertyName, Object propertyValue) thro
           }
           break;
 
+        case ENABLE_PUT_GET:
+          if (propertyValue != null) {
+            enablePutGet = getBooleanValue(propertyValue);
```

Comment:
can we use setter `setEnablePutGet` instead?

Reply:
sure. will do.

Reply:
Done.